packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx

```diff
@@ -1,38 +1,76 @@
 /* tslint:disable */
 import * as React from 'react';
 /* tslint:enable */
-import { BaseComponent, KeyCodes, getId, getNativeProps, inputProperties, css } from '../../../../Utilities';
+import { KeyCodes, getId, getNativeProps, inputProperties, css } from '../../../../Utilities';
 import { FloatingPeoplePicker, IBaseFloatingPickerProps } from '../../../../FloatingPicker';
-import { ISelectedPeopleItemProps } from '../SelectedPeopleList';
 import { IExtendedPersonaProps } from '../SelectedPeopleList';
 import { IPeoplePickerItemState } from './ExtendedSelectedItem';
 import { IPersonaProps } from '../../../../Persona';
 
-import * as stylesImport from './EditingItem.scss';
-
-// tslint:disable-next-line:no-any
-const styles: any = stylesImport;
-
-export interface IEditingSelectedPeopleItemProps extends ISelectedPeopleItemProps {
```

Comment:
Technically this is exported but I don't see it in the API file so it doesn't appear to be exported by default. I wonder though if we should leave it for backwards compatibility and deprecate it:
```tsx
/**
 * @deprecated Use IEditingItemProps
  */
export IEditingSelectedPeopleItemProps extends IEditingItemProps {};
```